apps/backend/src/db/ddb.ts

```diff
@@ -10,7 +10,11 @@ const client = new DynamoDB({
 });
 
 // Create DynamoDB DocumentClient
-const documentClient = DynamoDBDocument.from(client);
+const documentClient = DynamoDBDocument.from(client, {
+    marshallOptions: {
+        removeUndefinedValues: true
```

Comment:
Maybe we can add a quick snippet detailing why this setting was flipped.

```suggestion
        /**
         * Some JSON properties may exist and be undefined; DDB will throw an error unless this is true.
         * @example { "property": undefined }
         */
        removeUndefinedValues: true
```